### PR TITLE
Small fixes for RIPs 42, 46 and 47

### PR DESCRIPTION
**Pull request by knoshua**

### CODE REVIEW

File: RPIPs/RPIP-46.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ Some example concrete guidelines:
   - Eg, rETH demand can be increased by spending more RPL on marketing or partner incentives; that RPL can be sourced by increasing RPL inflation. This is beneficial because it allows targeted intervention to spur rETH demand.
   - Eg, rETH demand can be increased by increasing `reth_share` alongside a counterbalancing decrease to `voter_share`
   - Eg, rETH demand can be increased by increasing `reth_share` _and_ NO supply can be simultaneously decreased by a counterbalancing decrease to `no_share` 
-- If the NO queue is continuously over 1000 deposits for 4 weeks and the trend is upwards, the pDAO should take action to decrease the supply of NOs.
+- If the NO queue is continuously over 1000 deposits for 4 weeks and the trend is upwards, the pDAO should take action to decrease the supply of NOs
```
Comment (LongForWisdom) [inline RPIPs/RPIP-46.md:146]:
```suggestion
- If the combined node operator queues are continuously over 1000 deposits for 4 weeks and the trend is upwards, the pDAO should take action to decrease the supply of node operators
```

I think we should drop NO as an abbreviation. It's already caused confusion once. Reworded to cover multiple queues... Maybe that can be done more elegantly.

Comment (LongForWisdom) [reply RPIPs/RPIP-46.md:146]: Eh, maybe just disregard at this stage.

File: RPIPs/RPIP-46.md [hunk 2]
```diff
@@ -163,7 +163,7 @@ Some example concrete guidelines:
   - Eg, rETH demand can be increased by increasing `reth_share` alongside a counterbalancing decrease to `new_share`
   - Eg, NO supply can be decreased by reducing `no_share` alongside a counterbalancing increase to `new_share`
   - Eg, rETH demand can be increased by increasing `reth_share` _and_ NO supply can be simultaneously decreased by a counterbalancing decrease to `no_share`
-- If the NO queue is continuously over 1000 deposits for 4 weeks and the trend is upwards, the pDAO should take action to decrease the supply of NOs.
+- If the NO queue is continuously over 1000 deposits for 4 weeks and the trend is upwards, the pDAO should take action to decrease the supply of NOs
```
Comment (LongForWisdom) [inline RPIPs/RPIP-46.md:166]: See comment above.

**Review by LongForWisdom — COMMENTED** [review]
Minor comments that we should probably just ignore. Looks good otherwise.

**Comment by orangesamus** [comment]
:+1:

File: RPIPs/RPIP-46.md (hunk 1 above)
Comment (orangesamus) [reply RPIPs/RPIP-46.md:146]: Could go through a separate PR and fix all "NO" to "node operator"

**Pull request merged**